### PR TITLE
[timeline] various updates

### PR DESCRIPTION
**Pull request by chejennifer**
- plot zero values [issue #1030](https://github.com/datacommonsorg/website/issues/1030)
- link to browser page when click on chip [issue #1032](https://github.com/datacommonsorg/website/issues/1032)
- make tooltip labeling consistent with legend [issue #1032](https://github.com/datacommonsorg/website/issues/1033)

plot zero values:
Screen Shot 2021-07-15 at 9 55 34 AM

make tooltip labelling consistent with legend:
Screen Shot 2021-07-15 at 9 54 17 AM

### CODE REVIEW

File: static/js/chart/draw.ts [hunk 1]
```diff
@@ -159,7 +160,8 @@ function showTooltip(
 function getTooltipContent(
   dataGroupsDict: { [place: string]: DataGroup[] },
   highlightedTime: number,
-  unit?: string
+  unit?: string,
+  statsVarInfo?: { [key: string]: StatVarInfo }
```
Comment (beets) [inline static/js/chart/draw.ts:164]: nit: rename to statVarInfo for (new) consistency?

**Review by beets — APPROVED** [review]
thx for the fixes!

File: static/js/chart/draw.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/chart/draw.ts]: done

**Comment by beets** [comment]
@googlebot: rescan

**Pull request merged**